mono/mini/llvm-jit.cpp

```diff
@@ -530,7 +530,11 @@ static void
 force_pass_linking (void)
 {
 	// Make sure the rest is linked in, but never executed
-	if (g_getenv ("FOO") != (char*)-1)
+	char *foo = g_getenv ("FOO");
+	gboolean ret = (foo != (char*)-1);
```

Comment:
When you have a moment, could you make the name a bit more clear so future editors don't mistake it for accidentally-committed debugging code and try to remove it?